overlord/ifacestate/udevmonitor/udevmon_test.go

```diff
@@ -90,6 +90,7 @@ E: MINOR=3
 E: MAJOR=0
 E: DEVNAME=ghi
 E: DEVTYPE=bzz
+__END__
```

Comment:
Ha :-)

Reply:
Hmm, interesting!

Reply:
I wonder if shell was happy because end-of-input or if this shows the test was really not using it at all and something else was masked?

Reply:
I played with this and it looks like indeed EOF is fine and no error - a very interessting find :)